phpBB/phpbb/db/migration/data/v33x/bot_update.php

```diff
@@ -0,0 +1,91 @@
+<?php
+/**
+ *
+ * This file is part of the phpBB Forum Software package.
+ *
+ * @copyright (c) phpBB Limited <https://www.phpbb.com>
+ * @license GNU General Public License, version 2 (GPL-2.0)
+ *
+ * For full copyright and license information, please see
+ * the docs/CREDITS.txt file.
+ *
+ */
+
+namespace phpbb\db\migration\data\v33x;
+
+class bot_update extends \phpbb\db\migration\migration
+{
+	static public function depends_on()
+	{
+		return ['\phpbb\db\migration\data\v330\v330'];
+	}
+
+	public function update_data()
+	{
+		return [
+			['custom', [[&$this, 'add_duckduckgo_bot']]],
```

Comment:
Any particular reason why this object is passed by reference?
Looks like a copy and paste and not providing any actual functionality.

Reply:
> Looks like a copy and paste and not providing any actual functionality.

Indeed.